AngularJS/js/controllers/dashboardController.js

```diff
@@ -1,7 +1,10 @@
 var cont=app.controller('dashboardController', ['$scope', function($scope) {
-  $scope.query = {}
-  $scope.queryBy = '$'
-  $scope.books = [
+  this.filter = '';
+  this.filterType = '';
+  this.refreshBooks = function(){
```

Comment:
I think it's more like `filterBooks` than `refresh`. And leave a space after the parenthesis `function() {`